### DISCUSSION TITLE
Docs: Mention how to add custom buttons?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#3799] [General]
Could be an idea to add a section in the Buttons reference page on how to add custom buttons (custom colors), similar on how to add custom admonitions.
Granted, admonitions are slightly more complicated, but it could still be a nice improvement to have a small section explaining how to actually add your own button styles to use.

**Comment by squidfunk** [comment]
You can add another CSS selector by using the Attribute Lists extension – that's exactly what buttons are already using, and add an additional stylesheet to define a rule with the given selector to override button colors. Custom admonitions touch more classes, which is why they're documented separately. Customizing buttons is simple with CSS.

Please note that the official documentation is not a tutorial on web development. It only explains the most common cases.
